web-client/integration-tests/journey/admissionsClerkSearchesForPractitionersByName.ts

```diff
@@ -1,23 +1,11 @@
 import { ADVANCED_SEARCH_TABS } from '../../../shared/src/business/entities/EntityConstants';
```

Comment:
Since we have agreed to deprecate our cerebral integration tests in favor of cypress tests, I would suggest that instead of fixing this test we add coverage in cypress for whatever is tested in this test file that isn't already tested by our cypress integration and smoketests and then remove this test. What do you all think?

Reply:
Will do this while being tested.